unifiedWSHandler.js

```diff
@@ -180,167 +173,161 @@ const trackedAlerts = [
+        \n<:TR:818988588049629256> **TR**: ${terObj[continent].tr}  |  ${trPc}%`);
+    }
+    else if(showTerritory){
+        const vsPc = Number.parseFloat(payload.faction_vs).toPrecision(3);
+        const ncPc = Number.parseFloat(payload.faction_nc).toPrecision(3);
+        const trPc = Number.parseFloat(payload.faction_tr).toPrecision(3);
+        sendEmbed.addField('Territory Control', `\
+        \n<:VS:818766983918518272> **VS**: ${vsPc}%\
+        \n<:NC:818767043138027580> **NC**: ${ncPc}%\
+        \n<:TR:818988588049629256> **TR**: ${trPc}%`);
+    }
+    const  rows = await pgClient.query("SELECT a.channel, c.Koltyr, c.Indar, c.Hossin, c.Amerish, c.Esamir, c.Oshur, c.Other, c.autoDelete, c.territory, c.nonTerritory\
+    FROM alerts a LEFT JOIN subscriptionConfig c on a.channel = c.channel\
+    WHERE a.world = $1;", [server.toLowerCase()]);
+    Promise.allSettled(rows.rows.map(async row => {
+        if(row[continent.toLowerCase()] === undefined){
+            // If config is not successfully set then display alert and attempt to initialize config
+            config.initializeConfig(row.channel, pgClient);
+        }
+        else if(!row[continent.toLowerCase()] || (showTerritory && !row['territory']) || (!showTerritory && !row['nonterritory'])){
+            // Skip alerts configured to not show
+            return;
+        }
+        try {
+            const resChann = await discordClient.channels.fetch(row.channel);
+            if (resChann.type === 'DM'){ // DM
+                const messageId = await messageHandler.send(resChann, {embeds: [sendEmbed]}, "Alert notification");
+                if(messageId != -1 && trackedAlerts.indexOf(Number(payload.metagame_event_id)) > -1){
+                    pgClient.query("INSERT INTO alertMaintenance (alertID, messageID, channelID) VALUES ($1, $2, $3);", [`${payload.world_id}-${payload.instance_id}`, messageId, row.channel])
+                        .catch(err => {console.log(err);});
+                }
+                if(messageId != -1 && row.autodelete === true){
+                    const in50minutes = new Date((new Date()).getTime() + 3000000);
+                    const in95minutes = new Date((new Date()).getTime() + 5700000);
+                    if(['Indar', 'Hossin', 'Esamir', 'Amerish', 'Oshur'].includes(continent) && response.name.indexOf("Unstable Meltdown") == -1){
+                        pgClient.query("INSERT INTO toDelete (channel, messageID, timeToDelete) VALUES ($1, $2, $3)", [row.channel, messageId, in95minutes])
+                        .catch(err => {console.log(err);});
+                    }
+                    else{
+                        pgClient.query("INSERT INTO toDelete (channel, messageID, timeToDelete) VALUES ($1, $2, $3)", [row.channel, messageId, in50minutes])
+                        .catch(err => {console.log(err);});
+                    }    
+                }
+                else if(messageId == -1){
```

Comment:
Explicitly check permissions/deleted channel instead of just failed message

Reply:
Saw that this is DM whoops.  Same as above DM comment then

Reply:
Same as above